garden_ai/base_image_dockerfiles/Dockerfile_jupyter_tf_3.10

```diff
@@ -7,9 +7,7 @@ RUN conda create -y --name env
 RUN pip install "dill==0.3.5.1"
 RUN pip install jupyter notebook
 
-RUN pip install "tensorflow==2.14.0"
 RUN pip install "tensorboard==2.15.0"
-RUN pip install "tensorboard-data-server==0.7.1"

```

Comment:
tensorboard-data-server is a dependency of tensorflow. So it's still in the image, we just don't need to specify it